features/share/graphql/README.md

```diff
@@ -0,0 +1,45 @@
+<p align="center">
+  <a href="https://vendia.net/">
+    <img src="https://raw.githubusercontent.com/vendia/examples/main/vendia-logo.png" alt="vendia logo" width="100px">
+  </a>
+</p>
+
+# GraphQL in Vendia Share
+
+[Vendia Share](https://share.vendia.net) allows users to interact with their Universal Application (Uni) nodes through the use of managed [GraphQL](https://graphql.org/) APIs.
+
+## What is GraphQL?
+
+> GraphQL is a query language for your API, and a server-side runtime for executing queries using a type system you define for your data. GraphQL isn't tied to any specific database or storage engine and is instead backed by your existing code and data.
+
+> --[Introduction to GraphQL](https://graphql.org/learn/)
+
+Simply put, GraphQL is a _specification_ that defines how developers can query data. The specification does not provide _implementation details_, where the GraphQL resolvers must be written in a certain language and persist data to a certain datastore. Indeed, there are examples of GraphQL clients and servers written in a [number of programming languages](https://graphql.org/code/) that interact with any number of datastores.
```

Comment:
```suggestion
Put another way, GraphQL is a _specification_ that defines how developers can query data. The specification does not provide _implementation details_, where the GraphQL resolvers must be written in a certain language or persist data to a certain datastore. Indeed, there are examples of GraphQL clients and servers written in a [number of programming languages](https://graphql.org/code/) that interact with any number of datastores.
```

Reply:
We talk about GraphQL resolvers in this paragraph but we never really define that that means. I think having a quick definition would be nice.